src/transaction/log_impl.h

```diff
@@ -714,29 +714,40 @@ struct log_global
 
   // *INDENT-OFF*
   cublog::meta m_metainfo;
-  cublog::prior_sender m_prior_sender;
+private:
 #if defined (SERVER_MODE)
+  std::unique_ptr<cublog::prior_sender> m_prior_sender = nullptr;
   std::unique_ptr<cublog::prior_recver> m_prior_recver = nullptr;
-#endif // SERVER_MODE = !SA_MODE
+#endif /* SERVER_MODE = !SA_MODE */
+  // *INDENT-ON*
 
-  private:
-    std::mutex m_ps_lsa_mutex;
-    std::condition_variable m_ps_lsa_cv;
-    std::atomic<bool> m_ps_lsa_up_to_date;
-    LOG_LSA m_ps_consensus_flushed_lsa; // The quorum (number of the majority) of PS have done flushing log recrods until this.
+private:
+  // *INDENT-OFF*
+  std::mutex m_ps_lsa_mutex;
+  std::condition_variable m_ps_lsa_cv;
+  std::atomic<bool> m_ps_lsa_up_to_date;
+  LOG_LSA m_ps_consensus_flushed_lsa;	// The quorum (number of the majority) of PS have done flushing log recrods until this.
+  // *INDENT-ON*
 
-  public:
-  log_global ();
-  ~log_global ();
+public:
+    log_global ();
+   ~log_global ();
```

Comment:
```suggestion
  log_global ();
  ~log_global ();
```

Reply:
I tried that. The `indent` too moves it back. Maybe because `indent` tool does not know dtor syntax as it only knows pure-C. I will except from indentation.